app/lib/search/backend.dart

```diff
@@ -79,18 +80,33 @@ class SearchBackend {
       throw RemovedPackageException();
     }
 
-    final pv = (await _db.lookup<PackageVersion>([p.latestVersionKey])).single;
+    final pv = await _db.lookupValue<PackageVersion>(p.latestVersionKey,
+        orElse: () => null);
     if (pv == null) {
       throw RemovedPackageException();
     }
 
     final analysisView =
         await analyzerClient.getAnalysisView(packageName, pv.version);
 
+    // Extend tags with `-in-prerelease` for prerelease version.
```

Comment:
```suggestion
    // Find tags from latest prerelease (if there one)
    // This allows searching for tags with `<tag>-in-prerelease`.
    // Example: `is:null-safe-in-prerelease`, or `platform:android-in-prerelease`
```
This way I think we can better remember what this bit of code does.